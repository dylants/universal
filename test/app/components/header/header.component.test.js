import React from 'react';
import renderer from 'react-test-renderer';
import { StaticRouter } from 'react-router';

import Header from '../../../../app/components/header/header.component';

describe('The Header component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(
      <StaticRouter context={{}}>
        <Header />
      </StaticRouter>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
