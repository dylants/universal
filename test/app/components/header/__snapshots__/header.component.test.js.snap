// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Header component should render correctly 1`] = `
<header>
  <div
    className="links"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="/page1"
      onClick={[Function]}
    >
      Page 1
    </a>
    <a
      href="/page2"
      onClick={[Function]}
    >
      Page 2
    </a>
  </div>
</header>
`;
